lustre_exporter_test.go

```diff
@@ -137,1159 +186,1167 @@ func blacklisted(blacklist []string, metricName string) bool {
 }
 
 func TestCollector(t *testing.T) {
-	targets := []string{"OST", "MDT", "MGS", "MDS", "Client", "Generic", "LNET"}
+	targets := []string{"MGS", "OST", "MDT", "MDS", "Client", "Generic", "LNET"}
```

Comment:
I switched them for testing purposes (was impossible to figure out what I was doing wrong when testing against the OST data first). I can switch it back.